docs/other/install-gwms-frontend.md

```diff
@@ -0,0 +1,983 @@
+
+Monitoring Web pages
+--------------------
+
+You should be able to see the jobs also in the GWMS monitoring pages that are made available on the Web: **`http://gwms-frontend-host.domain/vofrontend/monitor/`**
+
+Troubleshooting
+===============
+
+File Locations
+--------------
+
+|  File Description  |                                                             File Location                                                    |
+|:------------------:|:----------------------------------------------------------------------------------------------------------------------------:|
+| Configuration file |                                                    /etc/gwms-frontend/frontend.xml                                           |
+|        Logs        |                                                        /var/log/gwms-frontend/                                               |
+|   Startup script   |                                                       /etc/init.d/gwms-frontend                                              |
+|    Web Directory   |                                                    /var/lib/gwms-frontend/web-area                                           |
+|      Web Base      |                                                    /var/lib/gwms-frontend/web-base                                           |
+|  Web configuration |                                                 /etc/httpd/conf.d/gwms-frontend.conf                                         |
+|  Working Directory |                                                  /var/lib/gwms-frontend/vofrontend/                                          |
+|     Lock files     | /etc/init.d/gwms-frontend/vofrontend/lock/frontend.lock /etc/init.d/gwms-frontend/vofrontend/group\_\*/lock/frontend.lock |
+|    Status files    |                               /var/lib/gwms-frontend/vofrontend/monitor/group\_\*/frontend\_status.xml                       |
+
+
+!!! note
+    `/var/lib/gwms-frontend` is also the home directory of the `frontend` user
+
+Certificates brief
+------------------
+
+Here a short list of files to check when you change the certificates. Note that if you renew a proxy or certificate and the DN remains the same no configuration file needs to change, just put the renewed certificate/proxy in place.
+
+|              File Description             |                               File Location                               |
+|:-----------------------------------------:|:-------------------------------------------------------------------------:|
+|             Configuration file            |                      /etc/gwms-frontend/frontend.xml                      |
+|         HTCondor certificates map         |                   /etc/condor/creds/condor\_mapfile (1)                   |
+|        Host certificate and key (2)       | /etc/grid-security/hostcert.pem&lt;br/&gt; /etc/grid-security/hostkey.pem |
```

Comment:
Get rid of the `&lt;br/&gt;` -- it doesn't work. split this into two rows instead.